# Sample NMEA parameter configuration

nmea:
  set: true
  version: 65
  num_sv: 8
  sv_numbering: 1
  compat: true
  consider: true
  limit82: true
  high_prec: false
  filter:
    pos: true
    msk_pos: true
    time: true
    date: true
    gps_only: true
    track: true
  gnssToFilter:
    gps: false
    sbas: true
    qzss: true
    glonass: false
    beidou: true
  main_talker_id: 1
  gsv_talker_id: 1
  bds_talker_id: [0,0]